tests/ci/ci_config.py

```diff
@@ -410,7 +410,9 @@ class CI:
             num_batches=6,
         ),
         JobNames.INTEGRATION_TEST_TSAN: CommonJobConfigs.INTEGRATION_TEST.with_properties(
-            required_builds=[BuildNames.PACKAGE_TSAN], num_batches=6
+            required_builds=[BuildNames.PACKAGE_TSAN],
+            num_batches=6,
+            timeout=9000,  # the job timed out with default value (7200)
```

Comment:
This should be fixed by https://github.com/ClickHouse/ClickHouse/pull/69322 and https://github.com/ClickHouse/ClickHouse/pull/69324